tests/phpunit/tests/block-template-utils.php

```diff
@@ -523,4 +529,235 @@ public function test_wp_generate_block_templates_export_file() {
 		}
 		$this->assertTrue( $has_html_files, 'contains at least one html file' );
 	}
+
+	/**
+	 * Tests `_get_block_template_file_content()` with a file that is part of the current theme.
+	 *
+	 * This should store the file content in cache.
+	 *
+	 * @ticket 59600
```

Comment:
Can we add some `@covers ::_get_block_template_file_content` annotations to the tests?

Reply:
Done in https://github.com/WordPress/wordpress-develop/pull/5463/commits/fa387c8b2a8a9058e202b7bfc6ae3c8b82c829d0